camunda-formio-tasklist-vue/.eslintrc.js

```diff
@@ -12,21 +12,25 @@ module.exports = {
     ecmaVersion: 2020,
   },
   rules: {
-    "no-console": "warn",
+    "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
-    "vue/no-unused-components": "warn",
-    "sort-imports": "warn",
-    camelcase: "warn",
+    "vue/no-unused-components": "error",
+    "camelcase": "warn",
     "vue/valid-template-root": "warn",
-    indent: ["warn", 2],
-    eqeqeq: "warn",
+    "indent": ["warn", 2],
+    "eqeqeq": "warn",
     "handle-callback-err": "warn",
     "prefer-promise-reject-errors": "warn",
     "no-unused-vars": "warn",
     "no-undef": "warn",
     "@typescript-eslint/no-inferrable-types": "off",
     "@typescript-eslint/no-non-null-assertion": "off",
-    "space-before-function-paren": ["warn", "always"],
     "semi": ["warn", "always"],
+    "eol-last": ["error", "always"],
+    "block-spacing": "error",
+    "linebreak-style": "error",
+    "object-curly-newline": ["error", "always"],

```

Comment:
These changes has a lot of effect. Do lint all files with these changes before merging codebase.

Noticed 32 files needs to be changed on running `npm run lint --fix`

![image](https://user-images.githubusercontent.com/70306694/133044837-cc7c9a57-88d0-45df-85e8-f89268782d50.png)

Reply:
I have pushed the linted changes along with PR #91